### PR TITLE
Fix inclusion of DataCleansing.java and MD5 hashes.

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for broken doc build.

Running as a Quick Build: https://builds.cask.co/browse/CDAP-DQB347-1

### CODE REVIEW

File: cdap-docs/examples-manual/source/examples/data-cleansing.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ of the application are tied together by the class ``DataCleansing``:
 
 .. literalinclude:: /../../../cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java
    :language: java
-   :lines: 32-
+   :lines: 30-
```
Comment (anwar6953) [inline cdap-docs/examples-manual/source/examples/data-cleansing.rst:33]: Can you leave out this change? It was due to the change to DataCleansing.java, as in: https://github.com/caskdata/cdap/pull/8858.
I had reverted the change to DataCleansing.java, but am bringing it back in that PR.

Comment (awholegunch) [reply cdap-docs/examples-manual/source/examples/data-cleansing.rst:33]: Well, until that change comes back, and the two are sync'd the docs will show the code incorrectly. If you are bring the change back, then in that PR you must:
- change the Java file
- change the Doc file
- change the MD5 hash.

It's best if these are done together...

Comment (awholegunch) [reply cdap-docs/examples-manual/source/examples/data-cleansing.rst:33]: Short answer: no.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**